index.html

```diff
@@ -0,0 +1,198 @@
+              <span></span>
+              <span></span>
+              <span></span>
+            </button>
+          </div>
+          <div class="logo-container">
+            <a href="index.html">
+              <img src="./assets/img/logo-original.png" alt="Logo Psychological therapy Week 2023">
+            </a>
+          </div>
+          <nav class="navbar-menu">
+            <ul class="box_flex align_center">
+              <li><a href="about.html" class="nav-link">About</a></li>
+              <li><a href="#" class="nav-link">Program</a></li>
+              <li><a href="#" class="nav-link">Join</a></li>
+              <li><a href="#" class="nav-link">Sponsor</a></li>
+              <li><a href="#" class="nav-link">News</a></li>
+              <li><a href="#" class="btn_default primary_line_button" id="btn-contact">MH Campaigns</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section id="home-headline" class="min_ht_full">
+      <div class="section-image-bg home-section"></div>
+      <div class="container box_flex flex_col justify_center">
+        <div>
+          <p class="first-greeting">"Hello, Powerful Minds!"</p>
+          <h1 id="home-title" class="headline-title">Psychological Therapy Week Event <span class="year">2023</span>
+          </h1>
+        </div>
+        <div class="headline-description">
+          <p>
+            In HappyMind Foundation, we want to build a more habitable and happy place through psychology and sociology.
+            We are pleased to invite you to our Psychological assistance day for the entire community, where we
+            will be offering individual and group psychological therapy sessions for individuals of all ages, races,
+            sex, and social strata. We will also be carrying out different social activities so that you can
+            share with other people and release stress and problems. In this opportunity, we will be helping the
+            community of Quebec, Canada.
+          </p>
+        </div>
+        <div>
+          <p id="headline-date-details">2023.06.05(MON) ~ 09(WEN)</p>
+          <p id="headline-location-info">@McGill University, FSA ULaval and more.</p>
+        </div>
+      </div>
+    </section>
+    <section id="main-program">
+      <div class="section-shape-bg"></div>
+      <div class="container">
+        <h2 class="section-title">Main Program</h2>
+        <div id="program-items-boxes">
+          <div class="program-item box_flex justify_center align_center">
+            <div class="item-icon box_flex justify_center align_center">
+              <img src="./assets/img/mental-health.png" alt="Conference icon" />
+            </div>
+            <h3 class="item-title">A Better World</h3>
+            <p class="item-details">
+              Welcome talk on human rights and to introduce our collaborating specialists.
+            </p>
+          </div>
```

Comment:
Kindly adjust the design of the program cards to be as close to the design as possible. On the design, the description of the programs aren't as thick.

Your design:
![image](https://github.com/NitBravoA92/capstone-project/assets/20554583/bac025f7-9df8-411d-b541-f7013ae155a6)

Design Guide:
![image](https://github.com/NitBravoA92/capstone-project/assets/20554583/dcf7b24b-338a-48b1-9e8d-664ada7cf3c7)

Reply:
Hello @emmiiorji  how are you? Thank you very much for your review, it is very valuable and helps me improve the code of my project. However, I would like to know a little more about this observation. When you say that "the description of the programs aren't as thick." Do you mean the amount of text in the box (there is a lot of text) or that the text should be more centered, leaving more spaces to the left and right?

Reply:
Hi @NitBravoA92

Sorry, the right term is **bold**. The description on your design is bolder than appears on the design.

Reply:
Hello @emmiiorji, thank you very much for clarifying my doubt. You're definitely right, it's a little bolder. I will make all these suggested changes. I really appreciate all your suggestions, I know they will make my code much better.